### PR TITLE
Add SQLAlchemy-JSONField

### PR DESCRIPTION
**Pull request by xylar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

This package is a new dependency of [airflow](https://github.com/conda-forge/airflow-feedstock/pull/38)

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
Can this be noarch python?

File: recipes/sqlalchemy_jsonfield/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - setuptools
+    - setuptools_scm
+    - sqlalchemy
```
Comment (mariusvniekerk) [inline recipes/sqlalchemy_jsonfield/meta.yaml]:
```suggestion
```

File: recipes/sqlalchemy_jsonfield/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - typing >=3.6  # py<37
```
Comment (mariusvniekerk) [inline recipes/sqlalchemy_jsonfield/meta.yaml]:
```suggestion
```

File: recipes/sqlalchemy_jsonfield/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+test:
+  imports:
+    - sqlalchemy_jsonfield
+
+about:
+  home: https://github.com/penguinolog/sqlalchemy_jsonfield
+  license: Apache 2.0
```
Comment (mariusvniekerk) [inline recipes/sqlalchemy_jsonfield/meta.yaml]:
```suggestion
  license: Apache-2.0
```

**Comment by xylar** [comment]
@mariusvniekerk, thanks very much for the review (on Christmas!) and the helpful suggestions.  

I don't know for sure if this package is supported under windows and I didn't understand the CI failure there (I don't use Windows and conda-forge myself).

**Comment by xylar** [comment]
@mariusvniekerk, it seems that several feedstocks aren't getting generated, with errors like:
```
Traceback (most recent call last):
  File "/home/travis/miniconda/bin/conda-smithy", line 10, in <module>
    sys.exit(main())
  File "/home/travis/miniconda/lib/python3.7/site-packages/conda_smithy/cli.py", line 546, in main
    args.subcommand_func(args)
  File "/home/travis/miniconda/lib/python3.7/site-packages/conda_smithy/cli.py", line 252, in __call__
    ci_register.add_project_to_drone(owner, repo)
  File "/home/travis/miniconda/lib/python3.7/site-packages/conda_smithy/ci_register.py", line 173, in add_project_to_drone
    response.raise_for_status()
  File "/home/travis/miniconda/lib/python3.7/site-packages/requests/models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://cloud.drone.io/api/repos/conda-forge/sqlalchemy-jsonfield-feedstock
Traceback (most recent call last):
  File ".travis_scripts/create_feedstocks.py", line 183, in <module>
    subprocess.check_call(['conda', 'smithy', 'register-ci', '--without-appveyor', '--feedstock_directory', feedstock_dir] + owner_info)
  File "/home/travis/miniconda/lib/python3.7/subprocess.py", line 341, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['conda', 'smithy', 'register-ci', '--without-appveyor', '--feedstock_directory', '/tmp/tmpehf3qca3__feedstocks/sqlalchemy-jsonfield-feedstock', '--organization', 'conda-forge']' returned non-zero exit status 1.
```
Others (e.g. `packer`) are getting created just fine.  Just a guess, but maybe it relates to earlier errors with GitHub API having been accessed more than the allowed 5000 times in an hour?  Is this something you expect to resolve itself or is some kind of manual interference required?  I can ask about this on Gitter if you prefer.

**Pull request merged**